src/tests/system/tests/test_ipa.py

```diff
@@ -0,0 +1,84 @@
+"""
+SSSD IPA tests.
+
+:requirement: ipa
+"""
+
+from __future__ import annotations
+
+import time
+
+import pytest
+from sssd_test_framework.roles.client import Client
+from sssd_test_framework.roles.ipa import IPA
+from sssd_test_framework.topology import KnownTopology
+
+
+@pytest.mark.ticket(bz=1926622)
+@pytest.mark.topology(KnownTopology.IPA)
+def test_sssd_ipa_indicators(client: Client, ipa: IPA):
+    """
+    :title: Add support to verify authentication
+     indicators in pam_sss_gss
+    :setup:
+        1. Add pam_sss_gss configuration to /etc/sssd/sssd.conf
+        2. Add pam_sss_gss.so to /etc/pam.d/sudo
+        3. Restart SSSD
+        4. Enable SSSD debug logs
+    :steps:
+        1. Switch to 'admin' user
+        2. obtain Kerberos ticket and check that it
+           was obtained using SPAKE pre-authentication.
+        3. Create sudo configuration that allows an admin to
+           run SUDO rules
+        4. Try 'sudo -l' as admin
+        5. As root, check content of sssd_pam.log
+       `6. Check if acquired service ticket has req. indicators: 0
+       `7. Add pam_sss_gss configuration to /etc/sssd/sssd.conf
+       `8. Check if acquired service ticket has req.
+           indicators: 2
+    :expectedresults:
+        1. Should succeed
+        2. Should succeed
+        3. Should succeed
+        4. Should succeed
+        5. Should succeed
+        6. Should succeed
+        7. Should succeed
+        8. Should succeed
```

Comment:
Where is "check that it  was obtained using SPAKE pre-authentication." this tested?

```suggestion
        1. As admin user obtain kerberos ticket.
        2. Create sudo configuration that allows an admin to
             run SUDO rules and run 'sudo -l' as admin
        3. As root, check content of sssd_pam.log
        4. Add pam_sss_gss configuration to /etc/sssd/sssd.conf
        5. Obtrain ticket again and run 'sudo -l'
        6. As root, check content of sssd_pam.log
    :expectedresults:
        1. Ticket was obtained using SPAKE pre-authentication.
        2. The rules are shown
        3. It contains req. indicators: 0
        4. The rules are shown.
        5. It contains req. indicators: 2

```